netlify.toml

```diff
@@ -5,7 +5,9 @@
   command = "NX_VERBOSE_LOGGING=true yarn patch-package && yarn nx build website"
 [build.environment]
   NETLIFY_USE_YARN = "true"
-  YARN_FLAGS = "--ignore-scripts"
+  # TODO: adjust these once https://github.com/netlify/build-image/issues/612 is resolved
```

Comment:
That awkward moment when the upstream repo is archived without resolving the issue... 🤷